crds/kubeaddons_v1alpha1_addon.yaml

```diff
@@ -20,6 +20,10 @@ spec:
       properties:
         spec:
           properties:
+            kubernetes:
+              properties:
+                minversion: string
```

Comment:
`minVersion` or `minSupportedVersion`, same for `maxVersion`. We then should validate that as part of the Cluster Configuration CRD 🤔, we'll need to get the kubernetes installed version

Reply:
:+1: